CONTRACT.md

```diff
@@ -0,0 +1,98 @@
+# `cumulus-sled` Contract
+
+`cumulus-sled` is a command-line interface for preparing and outputting Cumulus Messages for Cumulus Tasks. This contract defines how libraries should call `cumulus-sled` to integrate a task into a Cumulus Workflow.
+
+Every Cumulus Task includes a business function. `cumulus-sled` and a language-specific library for interacting with `cumulus-sled` are required to integrate a business function as a Cumulus Task into a Cumulus Workflow.
+
+**NOTE: This should be updated to call `cumulus-sled.zip` once we have created that package.**
+
+All `cumulus-sled` functions below are invoked via command line:
+
+```
+python ./message/message.py loadRemoteEvent '<event_json>'
+python ./message/message.py loadNestedEvent '<event_json>' '<context_json>'
```

Comment:
Passing the input as command line arguments introduces a couple of challenges:  we need to make sure that we properly escape that JSON (so that it doesn't include quotes that could screw up the parsing of the command line) and it could result in some extremely long command line arguments.  I'm not sure if there's an upper limit on that, but it seems like a large workflow with large inputs could easily reach the tens of thousands or hundreds of thousands of characters.

It may be better if the input were passed into STDIN.  For the cases where there are a couple of arguments, the input to `message.py` would probably be something like this:

```JSON
{
  "event_json": <event_json>,
  "context_json": <context_json>
}
```